### ISSUE TITLE
Bug: API keys only able to be set by environment variables

### ISSUE THREAD

**Issue by micahlmartin** [harness-io/terraform-provider-harness#71]
The latest update to [harness-go-sdk@0.1.10](https://github.com/harness-io/harness-go-sdk) introduced a bug where you're only able to set the API keys for both current gen and next gen through environment variables.

**Comment by mattcharlton** [comment]
No change with 0.1.11 - same error.  

This is the provider config (working with previous versions)

provider "harness" {
  endpoint   = var.api_endpoint
  account_id = data.vault_generic_secret.harness.data.account_id
  api_key    = data.vault_generic_secret.harness.data.api_key
}

variable for the api endpoint has been updated in line with changes.

**Comment by micahlmartin** [comment]
@mattcharlton strange. Let me take a look.

**Comment by micahlmartin** [comment]
@mattcharlton Can you give this build a try?

```
terraform {
  required_providers {
    harness = {
      source = "harness-io/harness"
      version = "0.1.12-beta1"
    }
  }
}
```

**Comment by stan-yago** [comment]
Ran into this issue as well. Was seeing
Error: invalid config: ApiKey must be set %!s(<nil>)

  on provider.tf line 1, in provider "harness":
   1: provider "harness" {

downgrading or upgrading to version 0.1.12-beta1 works!

**Comment by micahlmartin** [comment]
Thanks for confirming @stan-yago . Just published the new version `0.1.12`.
